Lib/fontTools/varLib/instancer/__init__.py

```diff
@@ -1377,6 +1383,44 @@ def _isValidAvarSegmentMap(axisTag, segmentMap):
     return True
 
 
+def downgradeCFF2ToCFF(varfont):
+
+    # Save these properties
+    recalcTimestamp = varfont.recalcTimestamp
+    recalcBBoxes = varfont.recalcBBoxes
+
+    # Disable them
+    varfont.recalcTimestamp = False
+    varfont.recalcBBoxes = False
+
+    # Save to memory, reload, downgrade and save again, reload.
+    # We do this dance because the convertCFF2ToCFF changes glyph
+    # names, so following save would fail if any other table was
+    # referencing glyph names. The second save+load might not be
+    # necessary and might be removed in the future, but for now
+    # it is the tested and working way.
+
+    stream = io.BytesIO()
+    log.info("Saving CFF2 font to memory for downgrade")
+    varfont.save(stream)
+    stream.seek(0)
+    varfont = TTFont(stream, recalcTimestamp=False, recalcBBoxes=False)
+
+    convertCFF2ToCFF(varfont)
+
+    stream = io.BytesIO()
+    log.info("Saving downgraded CFF font to memory")
+    varfont.save(stream)
+    stream.seek(0)
+    varfont = TTFont(stream, recalcTimestamp=False, recalcBBoxes=False)
```

Comment:
It would be nice to verify now that the second save is not needed.

Reply:
Okay. I'll do some testing.

Reply:
Did some testing. It's currently necessary. See comments.

Reply:
Good to know.

Reply:
I might fight it again, but I think this is better than current situation at least.